packages/plugin-npm-cli/sources/commands/npm/whoami.ts

```diff
@@ -0,0 +1,51 @@
+import {Configuration, Ident, MessageName}              from '@berry/core';
+import {PluginConfiguration, StreamReport, structUtils} from '@berry/core';
+import {npmHttpUtils}                                   from '@berry/plugin-npm';
+import {Clipanion}                                      from 'clipanion';
+import {Readable, Writable}                             from 'stream';
+
+// eslint-disable-next-line arca/no-default-export
+export default (clipanion: Clipanion, pluginConfiguration: PluginConfiguration) => clipanion
+
+  .command(`npm whoami [-s,--scope SCOPE]`)
+  .describe(`display username`)
```

Comment:
Can you add the `categorize` call? cf: https://github.com/yarnpkg/berry/blob/master/packages/plugin-npm-cli/sources/commands/npm/publish.ts#L16